perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4272,36 +4274,48 @@ perun_policies:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   addAdmin_SecurityTeam_Group_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
 
   removeAdmin_SecurityTeam_User_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
 
   removeAdmin_SecurityTeam_Group_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Group
 
   addUserToBlacklist_SecurityTeam_User_String_policy:
     policy_roles:
       - SECURITYADMIN: SecurityTeam
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: User
```

Comment:
Adding the user to a blacklist seems like a similar process to banning a member from resource/Vo, where we agreed that the member being banned isn't the primary object and doesn't need to be critical. (The SecurityTeams "rules" are being edited, not the user themselves).